projects/1.20/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -0,0 +1,381 @@
+{
+    "item.electrodynamics.guidebook": "EEC 第一版",
+    "item.electrodynamics.laminatedcoil": "叠层铜线圈",
+    "item.electrodynamics.mechanicalvalve": "机械阀门",
+    "item.electrodynamics.plasticfibers": "塑料纤维",
+    "item.electrodynamics.fiberglasssheet": "玻璃纤维板",
+    "item.electrodynamics.platecopper": "铜板",
+    "item.electrodynamics.portablecylinder": "便携式气瓶",
+    "item.electrodynamics.pressuregauge": "压力表",
+    "item.electrodynamics.raworechromium": "粗铬铁矿",
+    "item.electrodynamics.raworefluorite": "氟石矿",
```

```suggestion
    "item.electrodynamics.raworefluorite": "粗氟石晶体",
```
Fluorite译作氟石不能算错，Crystal应译出